docs/usage/configuration-options.md

```diff
@@ -705,6 +705,11 @@ If you enabled `automerge` in the Renovate config, you can speed up the automerg
 Caution (fixed in GitLab >= 12.7): when this option is enabled it is possible due to a bug in GitLab that MRs with failing pipelines might still get merged.
 This is caused by a race condition in GitLab's Merge Request API - [read the corresponding issue](https://gitlab.com/gitlab-org/gitlab/issues/26293) for details.
 
+## gitLabIgnoreApprovals
+
+Ignore the default project level approval(s), so that Renovate bot can automerge its merge requests, without needing approval(s).
+Under the hood, it creates a MR-level approval rule where `approvals_required` is set to `0`.
```

Comment:
Should be documented that it needs gitlab auto merge.

Or should it also work with normal automerge?

Reply:
Actually, maybe for both

Reply:
Added note on this. Also, we're about to get rid of `gitLabAutomerge` here: #10958